Q/QpidProton/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+using BinaryBuilder, Pkg
+
+name = "QpidProton"
+version = v"0.40.0"  # Use the current stable version of Qpid Proton
+
+# Collection of sources required to complete build
+# "https://downloads.apache.org/qpid/proton/0.40.0/qpid-proton-0.40.0.tar.gz"
+
+sources = [
+    ArchiveSource(
+        "https://downloads.apache.org/qpid/proton/$(version)/qpid-proton-$(version).tar.gz",
+        "0acb39e92d947e30175de0969a5b2e479e2983bc3e3d69c835ee5174610e9636"
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/qpid-proton-*
+
+mkdir build && cd build
+cmake -B . \
+      -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DBUILD_PYTHON=OFF \
+      -DBUILD_GO=OFF \
+      -DBUILD_TESTING=OFF \
+      -DBUILD_EXAMPLES=OFF \
+      -DCMAKE_BUILD_TYPE=Release \
+      ..
+
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
https://buildkite.com/julialang/yggdrasil/builds/19329#01961c6a-5d8a-4b5f-af61-d7bea2141a46/689-5245
```
┌ Warning: /cache/build/yggy-amdci7-1/julialang/yggdrasil/Q/QpidProton/build/x86_64-linux-gnu/hr4668Lj/x86_64-linux-gnu-libgfortran5-cxx11/destdir/lib64/libqpid-proton-cpp.so.12.13.1 contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder.Auditor /cache/julia-buildkite-plugin/depots/e2fd9734-29d8-45cd-b0eb-59f7104f3131/packages/BinaryBuilder/86I4Q/src/auditor/compiler_abi.jl:259
```
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```